kuma/users/views.py

```diff
@@ -687,27 +670,21 @@ def form_valid(self, form):
         So, we need to manually commit the user to the db for it.
         """
         selected_email = form.cleaned_data["email"]
-        if form.other_email_used:
-            data = {
-                "email": selected_email,
-                "verified": False,
-                "primary": True,
-            }
-        else:
-            data = self.email_addresses.get(selected_email, None)
+        if selected_email not in self.email_addresses:
+            return HttpResponseBadRequest("email not the default suggested one")
```

Comment:
This is basically a solution to "hackers" not allowing that it said in the `<input type="hidden" name="email" value="{whatwegotfrom oauth}">` to be what's submitted. So, I don't care how rough the error is. Just needs to reject and not trigger a Sentry crash or bother with a UI pleasant form validation message.